test/functional/cursor_tests.js

```diff
@@ -4163,6 +4163,95 @@ describe('Cursor', function() {
     }
   });
 
+  it('Should properly kill a cursor', {
+    metadata: {
+      requires: {
+        topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'],
+        mongodb: '>=3.2.0'
+      }
+    },
+
+    // The actual test we wish to run
+    test: function() {
+      // Load up the documents
+      const docs = [];
+      for (let i = 0; i < 1000; i += 1) {
+        docs.push({
+          a: i
+        });
+      }
+
+      const configuration = this.configuration;
+      const client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
+      let cleanup = () => {};
+      let caughtError = undefined;
+
+      return (
```

Comment:
what's the intent in wrapping all of it in parentheses?

Reply:
prettier did that automatically :(